core/src/main/java/com/linecorp/armeria/server/annotation/Attribute.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.internal.server.annotation.DefaultValues;
+
+import io.netty.util.AttributeKey;
+
+/**
+ * Annotation for mapping a {@link AttributeKey} from {@link RequestContext} onto the following elements.
+ *
+ * <p>a parameter of an annotated service method. </p>
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+public @interface Attribute {
+
+    /**
+     * The name of the {@link AttributeKey} to bind to.
+     */
+    String value();
+
+    /**
+     * The class of the {@link AttributeKey} to bind to.
+     * If you make AttributeKey with AttributeKeys.valueOf(MyAttributeKeys.class, "INT_ATTR"),
+     * prefix should be 'MyAttributeKeys.class'.
+     * See, <a href="https://armeria.dev/docs/advanced-custom-attributes/">advanced-customer-attributes</a>.
```

Comment:
```suggestion
     * If you created an {@link AttributeKey} with {@code AttributeKeys.valueOf(MyAttributeKeys.class, "INT_ATTR")},
     * the {@code prefix} should be {@code MyAttributeKeys.class}.
     * See <a href="https://armeria.dev/docs/advanced-custom-attributes/">advanced-customer-attributes</a>.
```

Please also fix the incorrect link. Please try generating the web site by running `./gradlew site:develop` and get the correct URL of the page.

Reply:
I fixed `Attribute.class` and there don't seem to be any broken links.
You can check java docs below. However, `gradle` seems not create Classes in `internal` package, thus i have no idea whether if there is broken links in `internal` pacakage. 

<img width="843" alt="image" src="https://github.com/line/armeria/assets/90125071/d3891300-b515-4dd4-b631-b7bba9b4a6a5">

Armeria docs seems looks good as well!

<img width="639" alt="image" src="https://github.com/line/armeria/assets/90125071/96e777a2-ad91-4fd9-bd94-3c8cf0f44b70">